src/libs/actions/IOU.ts

```diff
@@ -468,6 +472,19 @@ function buildOnyxDataForMoneyRequest(
                   },
               },
 
+        {
+            onyxMethod: Onyx.METHOD.SET,
```

Comment:
I see that some are using SET, while some are using MERGE for optimistic data.
Can you confirm all onyxMethods are set properly?